### DISCUSSION TITLE
Add Cherry Trees from 1.20 snapshots. And things related to tree

### DISCUSSION THREAD

**Discussion by MrcreativschikMrlowyaw** [Roadhog360/Et-Futurum-Requiem#254] [Ideas]
I think this deserve to be added

**Comment by Roadhog360** [comment]
Things from non-release builds are no longer going to be considered.

**Comment by GregoriusT** [comment]
Wait isn't 1.20 supposed to come out soon? Why not wait until that is out before saying something like this? Pretty sure Mojang did not remove the Cherry Tree, did they? You made it sound like that at least. ;)

**Comment by Roadhog360** [comment]
Yeah but it's still in progress so I don't want to add it yet. I remember what happened when I tried to add copper while it was in the snapshot phases, and I'm not making the same mistake again.
